enterprise/payara.common/src/org/netbeans/modules/payara/common/CommonServerSupport.java

```diff
@@ -710,7 +712,9 @@ public boolean isRemote() {
             return ServerAdmin.<ResultString>exec(instance, new CommandRedeploy(
                     name, Util.computeTarget(instance.getProperties()),
                     contextRoot, properties, libraries,
-                    url != null && url.contains("ee6wc")), stateListener);
```

Comment:
What is `ee6wc` and `deployer:pfv`? Some comment would help do understand this.

Reply:
That's URI fragment, see here: https://github.com/apache/netbeans/blob/master/enterprise/payara.common/src/org/netbeans/modules/payara/common/ServerDetails.java#L44-L250
URI fragment is assigned to each version of Payara Server e.g: deployer:pfv4ee7, deployer:pfv5ee8 etc, where pfv4 stands for Payara Fish version 4 and ee7 stands for supported Java EE version. Hence here checking for Payara uri fragment instead of a specific version.